src/picoprobe_config.h

```diff
@@ -65,9 +75,11 @@
 #ifndef PICOPROBE_LED
 
 #ifndef PICO_DEFAULT_LED_PIN
-#error PICO_DEFAULT_LED_PIN is not defined, run PICOPROBE_LED=<led_pin> cmake
+// #error PICO_DEFAULT_LED_PIN is not defined, run PICOPROBE_LED=<led_pin> cmake
+#define PICOPROBE_LED 0     // To avoid compiler errors -- is there a better way to do this?
```

Comment:
Perhaps you want something like `#if !defined(PICO_DEFAULT_LED_PIN) && !defined(PICO_DEFAULT_WS2812_PIN)` ?